spec/haskell/src/SEL4/Object/SchedContext.lhs

```diff
@@ -643,7 +643,8 @@ This module uses the C preprocessor to select a target architecture.
 >     when (numDomains > 1) $ do
```

Comment:
In one of your other PRs on a different function, you included a test for `num_domains` in the abstract, but you haven't done that here. Even though the test is always true in the current specification, we should try to be consistent about this.

Reply:
Excellent point. I'll chase it down.

Reply:
Done.